test/remote/gateways/remote_xpay_test.rb

```diff
@@ -22,44 +22,11 @@ def setup
     }
   end
 
-  def test_successful_purchase
-    response = @gateway.preauth(@amount, @credit_card, @options)
+  ## Test for authorization, capture, purchase and refund requires set up through 3ds
+  ## The only test that does not depend on a 3ds flow is verify
+  def test_successful_verify
```

Comment:
❓ Question:

Just to confirm:

isn't worth adding a test that confirms that setting up the transaction works properly,? I mean, I know that is not possible to get a completely successful transaction here, but need to confirm any way that we are sending transaction data correctly, that we are handing errors properly and may that scrub code actually works.

Reply:
good catch, as soon as the nexi xpay sandbox works again I will add it

Reply:
done!